frontend/src/test/resources/projects/mini-better-files/mini-better-files/src/main/scala/Implicits.scala

```diff
@@ -0,0 +1,263 @@
+package better.files
```

Comment:
Can we put this in an independent repo under the Scala Center domain instead of checking in the files?

Reply:
@Duhemm Can you fix this? :smile:. I can try to merge this when this is done!